### ISSUE TITLE
14

### ISSUE THREAD

**Issue by smileysridhar708-lgtm** [smileysridhar708-lgtm/Sj#1]
import nltk
from nltk.corpus import stopwords
from sklearn.feature_extraction.text import CountVectorizer
from sklearn.naive_bayes import MultinomialNB
from sklearn.model_selection import train_test_split
from sklearn.metrics import accuracy_score

nltk.download('stopwords')

texts = ["I love this product", "Great service", "Very disappointed",
         "Worst experience", "Amazing quality", "Not worth money"]

labels = ["pos", "pos", "neg", "neg", "pos", "neg"]

stop_words = set(stopwords.words('english'))
cleaned = [" ".join([w for w in t.lower().split() if w not in stop_words]) for t in texts]

X = CountVectorizer().fit_transform(cleaned)

X_train, X_test, y_train, y_test = train_test_split(X, labels, test_size=0.3)

model = MultinomialNB()
model.fit(X_train, y_train)

print("Accuracy:", accuracy_score(y_test, model.predict(X_test)))
